packages/mui-material/test/typescript/moduleAugmentation/tableCellCustomProps.spec.tsx

```diff
@@ -15,6 +15,12 @@ declare module '@mui/material/TableCell' {
   }
 }
 
+declare module '@mui/material/TableCell' {
+  interface TableCellPropsVariantOverrides {
+    tableBody: true;
+  }
```

Comment:
We can move this in the above `TableCell` module declaration thus having only one module declared.